controllers/djob_controller_databricks.go

```diff
@@ -88,13 +92,44 @@ func (r *DjobReconciler) delete(instance *databricksv1alpha1.Djob) error {
 
 	jobID := instance.Status.JobStatus.JobID
 
+	_, err := getJob(r, jobID)
```

Comment:
Is there a reason, that I'm missing, you made this into two lines, instead of keeping it a one liner as it was before?

Reply:
personally I found the "all in one" a bit harder to read, but think the actual reason for this was because I originally had the code working differently which wouldn't work in the one line format and then refactored it to a way where it would